INSTALL.md

```diff
@@ -0,0 +1,89 @@
+# Installing device plugins to cluster
+
+## Install device plugins via a daemonset
+
+Each plugin can be installed via a daemonset. The install changes slightly based on the desired plugin. See install instructions per [plugin](README.md#plugins).
+
+Installing plugins via daemonsets deployes them to the ```default``` (or currently active) namespace. Use kubectl's ```--namespace``` argument to change the deployment namespace.
+
+## Install device plugins via device plugin operator
+
+A more advanced install method is via device plugin operator. There the plugins are installed by supplying device plugin CRDs which the operator then converts into plugin deployments. See installation instructions in the [operator README](cmd/operator/README.md#installation).
+
+Device plugins installed via device plugin operator are installed in the device plugin namespace ```inteldeviceplugins-system```.
+
+## Install with HELM charts
+
+Device plugins can also be installed to a cluster with our Helm charts. Helm chart leverages the device-plugin operator which depends on cert-manager and NFD. The chart project for the operator is at https://github.com/intel/helm-charts/tree/main/charts/device-plugin-operator. Individual plugin projects are under https://github.com/intel/helm-charts/tree/main/charts/.
+
+As with the method above, these steps will install device plugin operator and plugins under ```inteldeviceplugins-system``` namespace.
+
+### Installing HELM repositories
+
+```bash
+helm repo add jetstack https://charts.jetstack.io # for cert-manager
+helm repo add nfd https://kubernetes-sigs.github.io/node-feature-discovery/charts # for NFD
+helm repo add intel https://intel.github.io/helm-charts/ # for device-plugin-operator and plugins
+helm repo update
+```
+
+### Installing cert-manager
+
+```bash
+helm install --wait \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.11.0 \
+  --set installCRDs=true
+```
+
+NOTE: cert-manager install takes a while to complete.
+
+### Installing NFD
+
+```bash
+cat > tmp.yaml <<EOF
+master:
+  extraLabelNs: ["gpu.intel.com","sgx.intel.com"]
+  resourceLabels: ["gpu.intel.com/millicores","gpu.intel.com/memory.max","gpu.intel.com/tiles","sgx.intel.com/epc"]
```

Comment:
Don't other plugins provide resources that should be listed here?

Reply:
I couldn't find others. @mythi can you confirm?

Reply:
just GPU and SGX